docs/pages/docker.rst

```diff
@@ -90,6 +98,11 @@ To run the bot, in both cases of installation, use this command:
   Same as ``-v`` option, they **must** be set between ``-d`` flag and
   ``perchunpak/pingerbot``.
 
+.. warning::
+  Since version 0.4.0 the container runs in rootless mode, which means that 
+  if you use the sqlite version and an external volume you have to give rights 
+  to the folder. This can be done with the command ``chown -R 5000:500 <your volume folder>``.
```

Comment:
Add windows alternative please.

Reply:
Those who host on Windows must die. It has no alternatives for chown as far as I know.

Reply:
https://serverfault.com/questions/27332/how-to-take-ownership-of-files-from-the-command-line

Reply:
Docker for windows runs in wsl. WSL already have all essential commands for Linux.

Reply:
This auto handled by Docker on Windows, so I add not about this in https://github.com/PerchunPak/pinger-bot/pull/51/commits/aece8091cd9f2a1b073eb7fa3ea9158b5c66de3d.